cmd/derper/depaware.txt

```diff
@@ -110,21 +115,25 @@ tailscale.com/cmd/derper dependencies: (generated by github.com/tailscale/depawa
      💣 tailscale.com/net/tshttpproxy                                from tailscale.com/derp/derphttp+
         tailscale.com/net/wsconn                                     from tailscale.com/cmd/derper+
         tailscale.com/paths                                          from tailscale.com/client/tailscale
-     💣 tailscale.com/safesocket                                     from tailscale.com/client/tailscale
+     💣 tailscale.com/safesocket                                     from tailscale.com/client/tailscale+
         tailscale.com/syncs                                          from tailscale.com/cmd/derper+
         tailscale.com/tailcfg                                        from tailscale.com/client/tailscale+
+        tailscale.com/tailfs                                         from tailscale.com/client/tailscale
+        tailscale.com/tailfs/compositefs                             from tailscale.com/tailfs
+        tailscale.com/tailfs/shared                                  from tailscale.com/tailfs/compositefs+
+        tailscale.com/tailfs/webdavfs                                from tailscale.com/tailfs
```

Comment:
These derper dependency changes aren't good and need to be removed.

Be sure to review depaware.txt files as part of code review. What's why they exist.

/cc @oxtoacart @creachadair @dsnet